File: BeitPatissierServer/Models/Ingredient.cs
namespace BeitPatissierServer.Models
{
    public class Ingredient //חומר גלם
    {
        public int IngredientId { get; set; }
        public string Name { get; set; }
        public decimal PricePerUnit { get; set; } // מחיר ליחידת מידה
    }
}

File: beit-patissier-manage/src/models/Ingredient.ts
export class Ingredient {
    ingredientId: number;
    name: string;
    pricePerUnit: number;

    constructor(
        ingredientId: number,
        name: string,
        pricePerUnit: number
    ) {
        this.ingredientId = ingredientId;
        this.name = name;
        this.pricePerUnit = pricePerUnit;
    }
}
